src/components/IdenticonSelector.vue

```diff
@@ -196,7 +194,7 @@
         left: 0;
         height: 100%;
         width: 100%;
-        padding: 4rem;
+        padding: 4rem 3rem;
```

Comment:
```suggestion
        padding: 4rem 3rem;
        border-radius: 1rem;
```

Reply:
@danimoh You have, but it has no effect (anymore, maybe).